src/AVFoundation/AVVideoSettings.cs

```diff
@@ -307,11 +310,12 @@ public AVVideoSettingsCompressed (NSDictionary dictionary)
 
 		public AVVideoScalingMode? ScalingMode {
 			get {
-				return AVVideoSettingsUncompressed.ScalingModeFromNSString (GetNSStringValue (AVVideo.ScalingModeKey));
+				var val = GetNSStringValue (AVVideo.ScalingModeKey);
+				return val is not null ? AVVideoSettingsUncompressed.ScalingModeFromNSString (val) : null;
```

Comment:
Same: I think it's simpler to change `ScalingModeFromNSString` to take a nullable string (in which case this change would not be needed at all).

Reply:
Missed this change, fixing in this next commit!